tests/devices/test_default_qutrit.py

```diff
@@ -405,14 +401,34 @@ class TestExpval:
                 0,
                 [[0, -1j, 0], [1j, 0, 0], [0, 0, 0]],
             ),
+            (qml.GellMannObs, [1, 0, 0], 0, 1),
+            (qml.GellMannObs, [0, 0, 1], 0, 1),
```

Comment:
These two entries differ in state, but match in expected output, just want to confirm this is as expected?

Reply:
Yep, this is expected behaviour.